materials/sql_joins.md

```diff
@@ -46,3 +46,5 @@ SELECT species, scientificname
 FROM Main
 JOIN Species WHERE species = new_code;
 ```
```

Comment:
Each of these examples of `JOIN` can have a little explanation about what the code is doing or what kind of shortcut is being taken. I would expect to really only use / need the 'Multi-table join' and 'Multi-table join with abbreviations' examples.